src/style.css

```diff
@@ -0,0 +1,139 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.searh-bar-big {
+  margin: 20px 0;
+  font-family: Avenir;
+  width: 100%;
+  height: 70px;
+  font-weight: 300px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 0 62px;
+  /* background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: left;
+  background-position-x: left; */
+}
+
+.searh-bar-big::placeholder {
+  font-family: Avenir;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.searh-bar-big:hover {
+  width: 100%;
+  height: 70px;
+  font-weight: 300px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  /* background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: left; */
+}
+
+.searh-bar-big:focus {
+  width: 100%;
+  height: 70px;
+  font-weight: 300px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  /* background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: left; */
+}
+
+.searh-bar-small {
+  font-family: Avenir;
+  width: 100%;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 0 33px;
+  /* background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: left; */
+}
+
+.searh-bar-small:hover {
+  width: 100%;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  /* background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: left; */
+}
+
+.searh-bar-small:focus {
+  width: 100%;
+  height: 42px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  /* background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: left; */
+}
+
+.searh-bar-small::placeholder {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.label-class-big {
+  position: relative;
+}
+
+.label-class-big::before {
+  content: "";
+  background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
combine into `background` property shorthand

Reply:
When I combine all the properties into the background property, my icon disappears